sms_sponsorship/models/sms_child_request.py

```diff
@@ -337,3 +345,30 @@ def sms_reminder_cron(self):
         for request in sms_requests:
             request.with_context(lang=request.lang_code).send_step1_reminder()
         return True
+
+    @api.model
+    def sms_notification_unfinished_cron(self):
+        """
+        CRON job that sends mails weekly, which notify the staff that some SMS
+        Sponsorship are ongoing.
+        :return: True
+        """
+        nb_sms_requests = self.search_count([
+            ('date', '<', fields.Date.today()),
+            ('state', 'in', ['new', 'child_reserved', 'step1']),
+        ])
+
+        # send staff notification
+        notify_ids = self.env['staff.notification.settings'].get_param(
+            'new_partner_notify_ids')
+        if notify_ids:
```

Comment:
```suggestion
        if nb_sms_requests and notify_ids:
```
To avoid sending a notification when there are no requests.